src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AppComponent } from './app.component';
+import { LoginComponent } from './login/login.component';
+import { WelcomeComponent } from './welcome/welcome.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'welcome', component: WelcomeComponent },
+  { path: '', component: LoginComponent },
```

Comment:
let's also add other 'login' path with LoginComponent, so on logout we can route it to login page with clear link